vsintegration/src/FSharp.Editor/Options/IntelliSenseOptions.fs

```diff
@@ -0,0 +1,48 @@
+﻿namespace Microsoft.VisualStudio.FSharp.Editor.Options
+
+open System.Runtime.InteropServices
+open System.Windows
+open System.Windows.Controls
+open Microsoft.CodeAnalysis.Completion
+open Microsoft.VisualStudio.LanguageServices.Implementation.Options
+
+open Microsoft.VisualStudio.FSharp.Editor
+open System.Windows.Markup
+
+module IntelliSenseOptionPageStrings =
+    let Option_CompletionLists = SR.GetString "Completion_Lists"
+    let Option_Show_completion_list_after_a_character_is_typed = SR.GetString "Show_completion_list_after_a_character_is_typed"
+    let Option_Show_completion_list_after_a_character_is_deleted = SR.GetString "Show_completion_list_after_a_character_is_deleted"
+
+type IntelliSenseOptionPageControl(serviceProvider) as this =
+    inherit AbstractOptionPageControl(serviceProvider)
+
+    let ( *** ) (control : 'T when 'T :> IAddChild) (children: UIElement list) =
```

Comment:
it'd be useful to have move all the UI related helper functions to a module so the other options pages can eventually use them too